Tests/MapboxNavigationTests/SKUTests.swift

```diff
@@ -23,26 +23,27 @@ class SKUTests: XCTestCase {
         
         XCTAssertEqual(speechSkuToken?.skuId, SkuID.navigationUser.rawValue)
     }
-    
-    func testSKUTokensMatch() {
+
+    // NOTE: Disable due to `UIApplication.shared.delegate` is nil during SPM testing. 
```

Comment:
Is there an alternative way to present a view controller in this environment, or would we perhaps split out a separate XCUITest target (as tail work)?

Reply:
We can't run XCUITests in SPM, can we? I think I need to put `#if SWIFT_PACKAGE` instead of disabling. 

Adding to the list of tail work. Will get back to this soon.